subset/bacnet/bacnetTests/src/main/java/PicsTest.java

```diff
@@ -0,0 +1,100 @@
+import com.google.common.collect.Multimap;
+import com.serotonin.bacnet4j.LocalDevice;
+import com.serotonin.bacnet4j.RemoteDevice;
+import com.serotonin.bacnet4j.npdu.ip.IpNetwork;
+import com.serotonin.bacnet4j.service.unconfirmed.WhoIsRequest;
+import helper.*;
+
+import java.util.Map;
+
+public class PicsTest {
+
+  private Connection connection;
+  private BacnetValidation validator;
+  private BacnetPoints bacnetPoints = new BacnetPoints();
+  private String passedTestReport = "RESULT pass protocol.bacnet.pic\n";
```

Comment:
this smells bad

private String testName = "protocol.bacnet.pic";
private String passedTestReport = String.format("RESULT pass %s\n", testName);